src/App.jsx

```diff
@@ -1,26 +1,195 @@
 import { useState } from 'react'
+import { useEffect } from 'react'
 import './App.css'
 import './stylesheets/movielist.css'
 
 import MovieList from './components/MovieList'
 import Header from './components/Header'
 import Footer from './components/Footer'
 
+const baseDetailsUrl = 'https://api.themoviedb.org/3/movie/now_playing?language=en-US'; //url that we are sending fetch request to that returns array with movies now playing and details without page at end
+const baseSearchUrl = 'https://api.themoviedb.org/3/search/movie'; //url that is the request that returns movie results depending on a query parameter which we are getting from our SearchBar component
```

Comment:
Let's move all this urls etc to a CONSTANTS file or CONFIG file, that will make easier to keep track of them